lang_scala/parsing/Parser_scala_recursive_descent.ml

```diff
@@ -2362,7 +2362,7 @@ let importSelectors in_ : import_selector list bracket =
   inBracesOrNil (commaSeparated importSelector) in_
 
 (** {{{
- *  ImportExpr ::= StableId `.` (Id | `_` | ImportSelectors)
+ *  ImportExpr ::= StableId `.` (Id | `_` | ImportSelectors) | metavariable
```

Comment:
again, please add a comment because those {{{ }}} were supposed initially to be reference to the official Scala grammar
(I just copied those comments when I ported the official Scala parser from Scala to OCaml).